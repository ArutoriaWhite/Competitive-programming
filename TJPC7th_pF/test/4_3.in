1663670 957
2189 1
2589 0
3843 1
4711 0
7505 1
13262 0
13489 1
14623 0
15148 1
15336 1
18419 0
20164 1
20360 1
22341 1
23525 0
27028 1
27588 1
28274 1
31336 1
32467 0
33072 1
34661 1
34722 0
37650 1
37712 1
43273 0
45551 1
46018 1
47987 0
48427 0
54110 1
55335 0
57493 1
61945 0
62835 1
63603 0
64160 1
65041 0
65063 1
68952 1
69895 1
70786 0
72669 1
73439 1
76194 0
78008 1
87578 0
88802 0
89460 0
89712 1
90563 1
93947 1
95024 1
95343 0
95933 1
99208 0
103116 0
104530 1
106512 0
106777 1
106818 1
108412 1
108910 0
110349 0
111008 0
112166 0
113213 1
117288 1
117617 1
119194 0
119894 1
121233 1
124840 1
126561 1
128445 0
129010 0
129361 0
136352 0
138353 0
138475 1
139988 0
140487 0
141349 1
142180 0
143303 1
143714 0
145320 1
145734 0
145913 0
146963 0
150393 0
154455 0
155623 1
155840 0
157984 0
159562 0
160410 1
160828 0
160839 1
167407 0
167686 0
172281 0
173541 1
178553 1
178685 0
180357 0
181066 0
184815 0
185649 0
186241 0
190564 1
192318 1
192603 1
193143 1
194806 1
195529 0
196946 1
201555 1
207213 1
208083 0
208200 0
211514 0
211704 1
211914 0
212635 0
215846 1
216357 1
216787 0
218842 1
222723 0
224928 0
226564 1
227011 0
229082 0
232270 1
233135 0
233909 1
234543 0
236479 0
237857 1
238430 0
240640 0
241025 1
242524 0
242780 1
243988 0
245969 1
246141 1
248475 1
249529 0
253429 0
254034 1
259357 1
259553 0
263375 1
264640 1
267136 0
272433 0
274255 0
276166 0
276514 1
277567 0
278920 0
280171 0
281252 0
282238 1
283187 1
283377 1
286813 0
288887 0
292168 1
296128 1
298189 0
306850 0
308935 0
311811 1
313274 0
315792 1
316021 1
316065 0
317929 1
320420 1
320568 1
323752 1
324114 1
325368 0
327494 1
333331 0
336142 1
337466 0
340260 0
342081 1
342114 1
343745 0
346526 1
347584 0
348837 0
350956 0
354272 0
355760 1
355780 0
356471 1
357345 0
357631 0
361393 1
364369 1
365127 1
367706 0
368953 1
373889 1
374738 1
376136 0
379285 0
380082 1
380267 1
381563 1
381662 0
384178 0
384473 1
388128 0
389822 0
394312 1
394419 1
395816 1
396266 0
399927 1
400121 0
402213 1
402899 1
404964 0
406521 1
410257 1
410283 0
412163 1
415717 1
417101 0
420244 1
420426 1
421863 1
421981 0
425216 0
427633 0
428005 0
428203 0
428602 0
430773 1
432422 0
434959 0
435160 1
436247 0
437106 0
437360 1
438013 0
439357 0
441851 1
446965 0
447357 1
448163 1
448358 1
448502 1
448580 1
451802 0
452085 0
459177 0
459414 1
462681 1
465262 0
465889 1
467685 1
467752 1
469478 1
470335 1
470554 1
470694 0
475321 0
478280 1
479617 0
479883 0
483115 0
483359 1
486715 1
489268 1
492036 0
495194 0
496654 1
497677 0
498075 0
499441 0
500389 1
502984 1
503977 1
505924 1
507287 0
507754 0
508881 1
512274 0
512715 1
519887 1
521106 1
522063 1
522310 0
522937 0
523096 1
525111 0
526024 1
526228 0
526369 1
526647 1
528401 0
529580 0
530112 0
531024 0
532289 0
533229 1
538830 1
539126 1
539910 1
541286 0
541358 0
542835 1
543290 0
544069 1
545153 0
547416 0
547718 1
548741 0
551453 1
552064 0
553847 0
556189 1
560211 1
561419 1
562614 1
564303 1
565084 0
568763 1
568830 0
571058 1
571288 1
574502 0
575833 1
576995 0
576999 1
577983 1
578604 1
579492 1
583944 0
590167 1
593215 1
594721 1
596825 1
599194 0
600759 0
605517 0
606214 1
606338 0
606674 1
611420 0
612806 0
614782 1
615169 1
616690 1
616973 0
617910 1
618292 1
618318 0
621194 0
623523 0
630036 1
630552 0
631228 0
634282 1
641246 0
642917 1
643171 1
643275 0
649078 1
651842 0
652069 0
653170 1
654679 1
655891 1
658168 0
658655 1
662383 1
662900 0
663118 0
666618 1
667799 0
668546 0
669982 1
670491 1
671148 0
672240 1
673555 0
679101 0
682765 1
683242 1
683559 0
689293 0
689928 0
690284 0
696400 1
697650 0
699736 1
700325 1
702700 1
703308 1
703359 0
703961 1
708888 1
710658 1
711996 1
712948 1
714090 0
714576 1
714601 1
717623 1
721592 1
722320 0
723324 0
723529 0
726616 1
727873 1
729394 0
731264 1
739345 0
740572 0
741737 1
743107 0
744424 1
746388 1
746482 0
747829 0
756468 1
756892 0
759494 0
763749 0
766239 0
766490 1
767394 1
770740 0
771624 0
771943 1
780165 1
780192 1
781767 1
782133 1
783450 1
784198 0
787557 1
790104 1
792384 1
792499 1
792630 1
795002 0
795651 1
798752 1
799571 1
800445 1
801209 1
801216 1
802363 1
805569 0
806792 1
807193 0
808789 0
808983 1
809134 1
811802 0
811983 1
812840 0
816765 0
816830 0
818271 0
820137 0
822528 1
824632 1
826814 0
827080 1
827571 1
827958 1
828634 0
831543 1
836167 0
837368 0
837555 1
838038 1
838919 1
839362 1
840142 0
841659 0
841728 1
842726 0
844414 0
847641 1
850994 1
853032 0
856366 0
856625 1
857741 1
858792 0
859066 1
862907 0
868926 1
870801 0
871362 1
871640 1
873605 0
875614 0
876155 0
876223 0
876788 0
880711 0
882481 1
884445 0
886302 0
887785 1
887976 1
888158 0
889051 1
889924 0
890820 1
892378 0
893015 0
895116 1
896859 1
901147 1
902113 0
902174 1
902403 0
911624 1
914422 1
921123 1
922380 0
924962 1
925645 0
925932 1
926507 1
927389 0
934590 1
937039 1
939875 0
940777 1
941542 1
945512 1
948677 1
948771 1
953206 1
954790 0
954952 1
957522 0
959723 0
959934 0
959964 1
960396 0
962955 1
965984 0
967770 1
970717 0
970791 0
972508 0
973092 0
974719 0
975255 0
975326 0
976911 1
977468 0
977889 1
977900 0
978384 0
979977 0
980076 0
982541 1
983805 1
984012 0
984135 0
986407 1
988626 1
990476 0
991073 0
994180 0
996002 0
997569 0
999277 0
1000457 1
1001283 1
1002457 0
1007059 1
1009531 0
1011514 0
1012660 0
1013858 1
1013953 0
1014550 0
1015374 1
1019623 0
1020113 1
1020237 1
1020311 0
1020790 0
1021469 1
1022847 0
1023047 0
1023551 0
1023568 0
1023618 1
1023649 0
1025206 0
1029687 1
1030415 0
1031055 0
1031833 1
1032113 0
1034010 0
1035591 0
1036822 1
1037230 0
1037685 1
1038072 1
1038288 0
1039220 0
1040225 1
1043782 0
1045045 0
1046559 0
1046660 1
1050167 0
1050757 1
1052548 1
1054638 1
1054912 1
1060539 0
1060995 0
1064128 0
1069594 0
1070310 1
1072420 1
1076579 0
1076803 1
1079215 0
1079980 1
1080210 0
1082945 0
1084312 0
1085369 0
1086796 1
1088082 0
1092513 1
1095350 1
1095425 1
1095742 0
1097323 1
1097676 1
1097853 1
1098053 0
1098822 0
1099588 1
1100089 1
1100305 0
1105408 1
1106509 1
1108644 0
1112267 1
1113219 0
1113390 0
1113625 0
1116281 1
1118116 1
1119333 0
1120313 1
1121021 1
1122564 0
1128229 1
1130771 1
1133520 1
1133720 1
1135804 0
1136822 1
1136857 0
1137508 1
1140332 0
1142729 0
1144172 1
1144375 1
1145037 1
1147596 1
1147599 1
1148271 1
1149470 0
1155614 0
1155623 0
1156466 1
1157287 1
1159987 0
1160678 0
1163494 1
1167088 0
1171682 0
1173089 1
1176989 1
1178146 0
1180786 1
1182878 0
1182938 1
1184774 0
1186072 1
1186149 0
1190735 1
1191778 0
1194033 1
1204122 0
1209826 1
1210536 0
1211463 1
1213997 1
1217219 0
1219630 0
1221418 0
1221875 1
1222028 1
1223900 1
1224354 0
1228021 0
1231555 0
1233115 1
1236927 1
1237273 1
1237852 0
1238115 0
1241043 0
1242004 0
1242312 0
1245558 1
1247253 1
1252965 1
1253368 0
1260621 0
1267115 1
1268577 1
1270528 1
1271557 1
1273004 0
1273623 1
1275656 1
1279807 0
1280534 1
1281874 0
1281938 0
1282506 1
1282887 0
1283781 1
1284419 1
1284992 0
1288613 1
1288876 0
1289040 1
1289137 0
1292026 1
1292353 0
1293438 0
1295311 1
1298062 1
1298267 1
1298579 1
1298868 0
1299745 1
1300285 1
1302824 1
1303968 0
1305799 0
1307761 0
1312463 0
1312736 1
1313694 0
1315888 1
1323480 0
1323704 0
1326542 1
1327352 1
1328016 0
1328246 0
1329639 0
1329722 0
1330349 1
1331726 1
1332113 0
1335010 1
1337032 1
1350331 1
1352950 1
1353206 1
1354003 0
1357525 0
1357783 1
1358911 0
1358953 0
1359740 1
1359939 0
1362588 1
1364341 1
1372965 0
1379078 0
1384454 0
1386202 0
1386457 0
1387574 0
1389954 0
1394650 1
1395076 1
1395217 0
1396542 0
1398101 1
1400788 0
1406165 0
1406680 0
1407845 1
1408686 0
1409871 0
1409897 0
1414137 1
1416803 0
1416899 1
1417842 0
1418234 0
1419511 0
1421322 0
1424295 1
1425006 0
1426371 1
1427285 0
1427904 1
1430231 0
1430804 0
1431232 0
1432302 0
1433439 0
1433848 0
1435844 1
1437791 0
1438010 1
1438565 0
1438670 1
1441345 1
1442004 1
1443516 0
1443599 1
1447109 1
1448260 1
1450523 0
1453503 0
1453791 1
1454611 0
1454681 1
1454682 1
1454898 0
1463831 0
1465173 1
1465881 0
1470340 0
1470861 1
1471208 0
1472121 0
1473132 0
1474583 0
1475559 1
1478263 1
1478857 1
1481969 1
1485403 1
1485663 0
1485691 1
1486273 1
1489408 0
1489604 0
1498497 0
1499317 1
1499523 0
1501468 1
1502226 1
1505084 0
1505721 1
1505986 1
1507377 0
1509095 1
1509668 1
1511477 1
1514132 1
1514538 0
1514774 1
1516695 0
1518048 1
1518553 0
1520582 1
1526233 1
1528213 0
1529582 0
1533040 1
1534609 0
1535930 1
1537676 1
1540359 0
1542050 1
1542737 1
1546928 1
1550790 1
1553787 0
1554674 0
1559179 0
1562543 0
1564817 0
1565323 1
1566879 0
1568692 0
1568897 0
1571262 0
1571796 0
1571933 0
1572310 1
1573577 0
1584068 0
1585969 0
1589469 0
1592197 0
1598318 0
1599551 0
1602519 0
1604293 1
1606465 1
1606965 0
1613073 1
1616011 1
1616940 1
1619656 1
1621713 1
1626327 1
1626967 1
1628088 1
1628692 0
1629166 1
1630856 0
1631240 1
1631310 1
1631483 0
1631535 0
1633321 0
1638809 1
1641519 1
1646445 1
1646723 0
1648183 0
1650302 1
1650896 1
1651227 1
1656170 0
1656867 0
1657740 0
