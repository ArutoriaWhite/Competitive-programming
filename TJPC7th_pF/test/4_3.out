911624 1654220
